packages/core/test/foundry/optimistic-assertor/OptimisticAsserter.Lifecycle.t.sol

```diff
@@ -54,16 +59,58 @@ contract OptimisticAsserterLifecycle is Test {
         vm.stopPrank();
     }
 
-    function testBondBelowMinimum() public {
-        vm.expectRevert("Bond amount too low");
-        optimisticAssertor.assertTruthFor(
-            bytes(claimAssertion),
-            address(0),
-            address(0),
-            address(0),
-            defaultCurrency,
-            0,
-            0
-        );
+    function test_AssertionWithDispute() public {
+        vm.startPrank(TestAddress.account1);
+        defaultCurrency.allocateTo(TestAddress.account1, optimisticAssertor.defaultBond());
+        assert(defaultCurrency.balanceOf(TestAddress.account1) >= optimisticAssertor.defaultBond());
+        defaultCurrency.approve(address(optimisticAssertor), optimisticAssertor.defaultBond());
+
+        // Account1 asserts a false claim.
+        bytes32 assertionId =
+            optimisticAssertor.assertTruthFor(
```

Comment:
maybe we should use `assertTruthFor` only when testing with non-default parameters?

Reply:
Good point!